### DISCUSSION TITLE
Query by relation length

### DISCUSSION THREAD

**Discussion by franfiori29** [prisma/prisma#13924] [Q&A]
I know I can do a query like this to find products with reviews:

const products = await prisma.product.findMany({
      where: {  reviews: { some: {} } },
  })

 but is there a way to query for products that have MORE than x amount of reviews?

**Comment by austincrim** [comment]
Hey @franfiori29 👋,

This is currently not possible with Prisma.

Feel free to read through [this relevant issue](https://github.com/prisma/prisma/issues/6570) and provide any feedback. This helps our engineering teams prioritize!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
